src/msg/async/AsyncConnection.cc

```diff
@@ -839,10 +839,8 @@ void AsyncConnection::process()
 
           // note last received message.
           in_seq.set(message->get_seq());
-          ldout(async_msgr->cct, 10) << __func__ << " got message " << message->get_seq()
-                               << " " << message << " " << *message << dendl;
-	  ldout(async_msgr->cct, 1) << " == rx == " << message->get_source() << " " << message << " " << *message
-				    << dendl;
+	  ldout(async_msgr->cct, 1) << " == rx == " << message->get_source() << " got message "
+                                    << message->get_seq() << message << " " << *message << dendl;
```

Comment:
missing space between seq and pointer.

also i'd remote 'got message' to keep it more concise.. the lines are already very long with the prefix